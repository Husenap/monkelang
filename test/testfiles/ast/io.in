monke work main give number monke do
  monke want number age
  monke say age
monke done

