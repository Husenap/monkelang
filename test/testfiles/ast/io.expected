FuncDeclStmt: name:main return:number
  Params: 
  Body: 
    BlockStmt: 
      ReadStmt: type:number id:age
      PrintStmt: 
        IdUseExpr: age
